package.json

```diff
@@ -3,8 +3,11 @@
   "version": "5.0.0",
   "private": true,
   "dependencies": {
-    "@emotion/react": "latest",
-    "@emotion/styled": "latest",
+    "@devexpress/dx-react-chart": "^4.0.3",
```

Comment:
I don't want to use devexpress for this. They [aren't using a common open source](https://github.com/DevExpress/devextreme-reactive/blob/master/LICENSE.md) license like the MIT license. We need to be careful about introducing new packages. Please use something like [Chart.js](https://www.chartjs.org/docs/latest/charts/doughnut.html) which uses the MIT license.